android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsViewModel.kt

```diff
@@ -52,7 +53,7 @@ class SurveyQuestionsViewModel(
         }
     }
 
-    fun answerQuestion(questionAnswer: Pair<Int, Int>) {
-        Timber.d("question ${questionAnswer.first} - answer ${questionAnswer.second}")
+    fun answerQuestion(questionAnswer: Pair<Int, List<Int>>) {
```

Comment:
I think we need a DTO for `Pair<Int, List<Int>` data structure now. It's more complex and we don't know exactly what's inside.

Reply:
Yes, please review more on the UI parts, I updated this one for all type of answer views in [this pull request](https://github.com/suho/kmm-ic/pull/123)